cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -55,7 +55,8 @@ func NewAddCmd(o *AddOrUpdateOptions, flagsFactory cmdcore.FlagsFactory) *cobra.
 				[]string{"package", "repository", "add", "-r", "tce", "--url", "projects.registry.vmware.com/tce/main:0.9.1"}},
 		}.Description("-r", o.pkgCmdTreeOpts),
 		SilenceUsage: true,
-		Annotations:  map[string]string{cmdapp.TTYByDefaultKey: ""},
+		Annotations: map[string]string{cmdapp.TTYByDefaultKey: "",
+			cmdcore.PackageManagementCommandsHelpGroup.Key: cmdcore.PackageManagementCommandsHelpGroup.Value},
```

Comment:
Should these be `PackageRepoHelpGroup`?

Reply:
The only command in package help group is `package`

The commands under the `package` command group are split into management, authoring and repository commands

Reply:
I let this be because I imagine folks will use them while managing packages
I was not very sure about creating a `PackageRepositoryHelpGroup`

Reply:
`repo release` is a part of `PackageAuthoringHelpGroup` as well

Reply:
Yeah, I don't have any strong opinions on this.